### PR TITLE
Revert uppercasing keywords in tokenizer

### LINKED ISSUE #95: Bug: Column names equal to keywords are uppercased
I have the following Column definition in my Symfony project

```php
#[ORM\Entity]
class Queue {
   [...]

    #[ORM\Column(
        insertable: false,
        updatable: false,
        columnDefinition: 'SERIAL',
    )]
    private ?int $autoIncrement = null;

   [...]
}
```

When generating a migration without `--formatted` option the column name is `auto_increment`. But with `--formatted` the column name will be `AUTO_INCREMENT`.

The resulting column name should be lowercase.

Edit: The `doctrine/sql-formatter` version is 1.2.0
Could be related to https://github.com/doctrine/sql-formatter/pull/93

### PR DESCRIPTION
**Pull request by mvorisek**
revert #93
fix #95

as discussed in https://github.com/doctrine/sql-formatter/issues/95#issuecomment-2107227703

### CODE REVIEW

File: src/Tokenizer.php [hunk 1]
```diff
@@ -898,7 +898,7 @@ private function createNextToken(string $string, Token|null $previous = null): T
             ) {
                 return new Token(
                     Token::TOKEN_TYPE_RESERVED_TOPLEVEL,
-                    substr($upper, 0, strlen($matches[1])),
+                    substr($string, 0, strlen($matches[1])),
```
Comment (mvorisek) [inline src/Tokenizer.php:901]: There is something with the coverage, I belive this is tested by phpunit, please help.

File: phpunit.xml.dist [hunk 2]
```diff
@@ -7,8 +7,6 @@
          beStrictAboutOutputDuringTests="true"
          colors="true"
          cacheDirectory=".phpunit.cache"
-         requireCoverageMetadata="true"
-         beStrictAboutCoverageMetadata="true"
```
Comment (derrabus) [inline phpunit.xml.dist]: Unrelated change, please revert.

Comment (greg0ire) [reply phpunit.xml.dist]: Addressed in https://github.com/doctrine/sql-formatter/pull/105

Comment (mvorisek) [reply phpunit.xml.dist]: reverted here

**Review by greg0ire — APPROVED** [review]

**Comment by greg0ire** [comment]
Thanks @mvorisek
